src/Core/Components/Overlay/FluentOverlay.razor.ts

```diff
@@ -0,0 +1,57 @@
+export namespace Microsoft.FluentUI.Blazor.Overlay {
```

Comment:
My first time typescript. Please verify. I put this together by taking a look on the other typescript files in the project.